lang/cs/Org.Apache.REEF.Common/Services/ServiceConfiguration.cs

```diff
@@ -42,6 +46,8 @@ public class ServiceConfiguration : ConfigurationModuleBuilder
         [SuppressMessage("Microsoft.Security", "CA2104:Do not declare read only mutable reference types", Justification = "not applicable")]
         public static readonly OptionalParameter<IService> Services = new OptionalParameter<IService>();
 
+        public static readonly OptionalParameter<string> ClientConfigString = new OptionalParameter<string>();
```

Comment:
This reveals the workaround as a public field. Is there a way for us to hide this? The fact that we do serialize things into strings shouldn't leak to the user, as we probably will find a better way in the long run.